modules/pubmaticBidAdapter.js

```diff
@@ -709,6 +760,23 @@ function _handleEids(payload) {
     payload.user.eids = eids;
   }
 }
+// TODO: need a better way to identify mediaType from response.
+function _checkMediaType(adm, newBid) {
+  var admStr = '';
+  if (adm.indexOf('<VAST version=>') >= 0) {
```

Comment:
need to consider case-insensitive regex, as there will be whitespaces
version=> will not satisfy for version=3.0>